eng/pipelines/templates/steps/analyze.yml

```
@@ -115,6 +115,11 @@ steps:
       TestMarkArgument: ${{ parameters.TestMarkArgument }}
       AdditionalTestArgs: ${{parameters.AdditionalTestArgs}}
 
+  - template: ../steps/update_snippet.yml
```

```suggestion
  - template: /eng/pipeines/templates/steps/update_snippet.yml
```

More portable for copy/paste if you use absolute paths from root.